Jieun/services/userService.js

```diff
@@ -1,28 +1,39 @@
 const userDao = require("../models/userDao");
+const {
+  passwordValidationCheck,
+  emailValidationCheck,
+} = require("../utils/validation-check");
 const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
```

Comment:
```suggestion
const bcrypt = require("bcrypt");
const jwt = require("jsonwebtoken");

const userDao = require("../models/userDao");
const {
  passwordValidationCheck,
  emailValidationCheck,
} = require("../utils/validation-check");
```
모듈을 import 하는 순서에 대한 컨벤션은 built-in 모듈 - 외부 모듈 - custom 모듈 순서입니다!